numpy/linalg/tests/test_linalg.py

```diff
@@ -201,21 +233,16 @@ def __repr__(self):
     LinalgCase("hmatrix_1x1",
                np.random.rand(1, 1),
                None),
-]
+])
 
 
 #
 # Gufunc test cases
 #
+def _make_generalized_cases():
```

Comment:
Avoiding global namespace pollution - stops anything accidentally using these local variables in a test. Also handy for code folding, sort of